aries-site/src/pages/components/namevaluelist.mdx

```diff
@@ -85,6 +87,18 @@ A NameValuePair is a set of two related elements: a name, which identifies a dat
   <NameValueListSimpleExample />
 </Example>
 
+### Multiple two column lists on a single page 
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe"
+  height={{ min: 'small' }}
+  code={[
+  "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/NameValueListMultipleDefaultExample.js",
+  "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/data.js",
+]}
```

Comment:
```suggestion
]}
template
```

Think we should leverage the `template` prop here since it's a full page example. Will be better for alignment and spacing.